docs/advanced_usage.rst

```diff
@@ -217,7 +217,7 @@ using the identifier which it has assigned to each document. You will end up get
 files like ``0000123.pdf`` in your media directory. This isn't necessarily a bad
 thing, because you normally don't have to access these files manually. However, if
 you wish to name your files differently, you can do that by adjusting the
-``PAPERLESS_FILENAME_FORMAT`` configuration option.
+``PAPERLESS_FILENAME_FORMAT`` configuration option. Paperless adds the ending for example ``pdf``automatically. 
```

Comment:
I think technically the extension will be added based on file type, so it could be `.jpg`, `.png`, etc.  This should probably be a more generic statement to encompass this.

Reply:
```suggestion
``PAPERLESS_FILENAME_FORMAT`` configuration option. Paperless adds the correct
file extension e.g. ``.pdf``, ``.jpg`` automatically. 
```

Yea, also line is too long, how about this.